src/Microsoft.Azure.SignalR.Common/Utilities/ClaimsUtility.cs

```diff
@@ -129,6 +129,20 @@ public static ClaimsPrincipal GetUserPrincipal(this OpenConnectionMessage messag
             return GetUserPrincipal(message.Claims);
         }
 
+        public static IEnumerable<Claim> CreateUserClaims(string userId, IEnumerable<Claim> claims)
```

Comment:
why not reusing `BuildJwtClaims`?

Reply:
The claims type user id in  `BuildJwtClaims` is a customized type `Constants.ClaimType.UserId` while that in management SDK is `ClaimTypes.NameIdentifier`. Is it a breaking changes?